WooCommerce/Classes/POS/Presentation/Item Search/POSProductSearchable.swift

```diff
@@ -3,18 +3,25 @@ import enum Yosemite.POSItemType
 import protocol Yosemite.POSSearchHistoryProviding
 
 @available(iOS 17.0, *)
+
 final class POSProductSearchable: POSSearchable {
+    internal let itemListType: ItemListType
     private let itemsController: PointOfSaleSearchingItemsControllerProtocol
     private let searchHistoryProvider: POSSearchHistoryProviding
 
-    init(itemsController: PointOfSaleSearchingItemsControllerProtocol,
+    init(itemListType: ItemListType,
+         itemsController: PointOfSaleSearchingItemsControllerProtocol,
          searchHistoryProvider: POSSearchHistoryProviding) {
+        switch itemListType {
```

Comment:
Could we set `self.itemListType = itemListType` here?

Reply:
Definitely should https://github.com/woocommerce/woocommerce-ios/pull/15568/commits/67c0893c99cf510bc13f237c4e7856dfcb37b170